2-mandatory/4-restaurant.js

```diff
@@ -20,7 +20,17 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
+  orderBurger (balance){
```

Comment:
here the balance is actually the remaining balance so at some point we would not have any remaining balance left